packages/curve-ui-kit/src/widgets/Footer/Description.tsx

```diff
@@ -0,0 +1,57 @@
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import Link from '@mui/material/Link'
+import Typography from '@mui/material/Typography'
+
+import { LogoImg } from 'ui/src/images'
+
+const LogoImageSrc = (LogoImg as unknown as { src: string }).src
+const Logo = styled('img')({
+  width: 48,
+  height: 48,
+  margin: 8,
+  alt: 'Curve',
+})
+
+export const Description = () => (
+  <Box
+    sx={{
+      display: 'flex',
+      flexDirection: 'column',
+      maxWidth: '330px',
```

Comment:
Hardcoded width isn't ideal.. Can't we have it size automatically?
The parent already has a fixed size

@0xtutti it looks hardcoded at 328px in the design